dev/examples/controls/bar.js

```diff
@@ -234,6 +234,40 @@ export const renderBarTimeSeriesWithDateline = (id) => {
     );
     return barTimeDateline;
 };
+
+export const renderBarTimeSeriesWithEventline = (id) => {
+    const axisData = utils.deepClone(getDemoData(`#${id}`, "BAR_TIMESERIES"));
+    axisData.axis.x.ticks = {
+        values: tickValues,
+        format: "%a %b %e"
+    };
+    (axisData.eventline = [
+        {
+            color: "black",
+            style: {
+                strokeDashArray: "4,4"
+            },
+            value: new Date(2017, 12, 2).toISOString()
+        },
+        {
+            color: "black",
+            style: {
+                strokeDashArray: "2,2"
+            },
+            value: new Date(2017, 12, 3).toISOString()
+        }
+    ]),
+        (axisData.clickPassThrough = {
+            dateline: false
```

Comment:
Shouldnt this be `eventline`. Ensure `clickPassThrough` works properly.

Reply:
Since we don't have any onClick property I have removed clickPassThrough property.